src/stdlib/String.ts

```diff
@@ -24,5 +24,40 @@ class LCaseCallable extends Callable {
     }
 }
 
+class AscCallable extends Callable {
+    arity = {
+        required: 1,
+        optional: 0
+    }
+
+    call(interpreter: Interpreter, str: BrsString): BrsType {
+        if (str.value.length == 0)
+            return new Int32(0);
+        else
+            return new Int32(str.value[0].charCodeAt(0));
+    }
+}
+
+class ChrCallable extends Callable {
+    arity = {
+        required: 1,
+        optional: 0
+    }
+
+    call(interpreter: Interpreter, value: BrsType): BrsType {
+        if (value instanceof Int32) {
```

Comment:
Any particular reason to use an `instanceof` check vs. comparing to `value.kind === ValueKind.Int32`?  The TS compiler should give you the same type support, but at runtime we only have to do a numeric comparison.

It's not a blocker by any means &ndash; I'm just curious (and secretly hoping I haven't somehow broken those `value.kind` comparisons :sweat_smile:)

Reply:
Unfamiliarity with the type system.  Will change.